Assets/Scripts/MaterialReader.cs

```diff
@@ -864,6 +866,21 @@ public void ChangeWindowEmissionColor(Material material, WindowStyle windowStyle
             }
         }
 
+        /// <summary>
+        /// Removes from cache all materials that have not been accessed from the time in minutes defined
+        /// by <see cref="AssetCacheThreshold"/>. If equals to <c>0</c> assets are never removed from cache.
+        /// </summary>
+        internal void ValidateCache()
+        {
+            if (AssetCacheThreshold == 0)
+                return;
+
+            float time = Time.realtimeSinceStartup;
+            float threshold = AssetCacheThreshold * 60;
+            foreach (var item in materialDict.Where(x => time - x.Value.timeStamp > threshold).ToList())
+                materialDict.Remove(item.Key);
+        }
```

Comment:
Can you add some log, say "x materials expired"?

Reply:
I don't think this would be valuable information, the log is already a bit "spammy" at times...